Src/coffeescript/cql-execution/src/datatypes/uncertainty.coffee

```diff
@@ -5,15 +5,15 @@ module.exports.Uncertainty = class Uncertainty
     if obj instanceof Uncertainty then obj else new Uncertainty(obj)
 
   constructor: (@low = null, @high) ->
-    gt = (a, b) -> if a.constructor.name is 'DateTime' then a.after b else a > b
+    gt = (a, b) -> if a.constructor.name in ['DateTime','Quantity'] then a.after b else a > b
```

Comment:
I mentioned in the `Quantity` class that I'm not crazy about standardizing on the `after`, `sameOrAfter`, etc, method names.  But whatever names we standardize on, I think it might scale better if we just check for the presence of that function rather than checking the constructor name.  In other words, if an object has a `greaterThan` function, use it, else use `>`.